aws/DRVault-Single-Deployment.yaml

```diff
@@ -395,14 +395,30 @@ Resources:
 
           def handler(event, context):
             try:
-              response = boto3.client('ec2').describe_images(
-                  Owners=['138339392836', '284566481789', '923248178732', '604515549189'],
-                  Filters=[
-                    {'Name': 'tag:Cyberark_version', 'Values': [event['ResourceProperties']['Version']]},
-                    {'Name': 'tag:component', 'Values': [event['ResourceProperties']['Component']]},
-                    {'Name': 'tag:os', 'Values': [event['ResourceProperties']['OS']]},
-                  ],
-              )
+              ver= event['ResourceProperties']['Version']
+              major= int(ver.split(".")[0])
+              minor= int(ver.split(".")[1])
+              response = {
+                  "Images": []
+              }
+
+              # In release versions previous to 12.2, "os" filter didn't exist
+              os_filter = {} if ver in ['11.5','11.6','11.7','12.0','12.1'] else \
```

Comment:
better to put the array in a variable with meaningful name - then the logic is self explainatory
i.e.
```
var pasVerWithoutOsFilter = ['11.5','11.6','11.7','12.0','12.1']
os_filter = {} if ver in pasVerWithoutOsFilter else \
```

Reply:
done